common/testData/localeIdentifiers/localeDisplayName.txt

```diff
@@ -356,7 +356,7 @@ es-Cyrl-MX; Spaans (Cyrillies, Meksiko)
 hi-Latn; Hindi (Latyn)
 nl-BE; Nederlands (België)
 nl-Latn-BE; Nederlands (Latyn, België)
-zh-Hans-fonipa; Chinees (Vereenvoudig, FONIPA)
+zh-Hans-fonipa; Chinees (Vereenvoudig, fonipa)
```

Comment:
This change enables TestLocaleDisplay to pass. The all-caps version FONIPA presumably came from the code-fallback value for a path like `//ldml/localeDisplayNames/variants/variant[@type="FONIPA"]`. Since this PR removes that code-fallback, the value needs to come from elsewhere; it's not clear to me where it comes from. 
Maybe from the string "zh-Hans-fonipa" itself? This line in `TestLocaleDisplay` seems to be where it happens:

```
    String name = cldrFile.nameGetter().getNameFromParserOpt(ltp, nameOpt);
```

See `testInputLocales` in GenerateLocaleIDTestData.java. The value `Chinees (Vereenvoudig, fonipa)` seems no worse than `Chinees (Vereenvoudig, FONIPA)`.